README.md

```diff
@@ -74,6 +95,16 @@ final HCaptchaConfig config = HCaptchaConfig.builder()
 HCaptcha.getClient(this).verifyWithHCaptcha(config)...;
 ```
 
+Also to reduce hCaptcha start time, `setup` can be used:
+
+```java
+HCaptcha hCaptcha = HCaptcha.getClient(this).setup()
+```
+
+`setup` accepts the same arguments as `verifyWithHCaptcha`. Once `setup` is called, later you can call `verifyWithHCaptcha` with no arguments
+
+If `verifyWithHCaptcha` will be called with different arguments than `setup`, SDK will handle this and re-configure hCaptcha.
```

Comment:
```suggestion
If `verifyWithHCaptcha` is called with different arguments than `setup` the SDK will handle this by re-configuring hCaptcha.
```

Reply:
I would say that with a new config object it will "work as before", so no benefits, because it will create a fragment instance.

We added "load from the cache" to WebView, but it doesn't look related to this place in the documentation